src/libs/SearchUtils.ts

```diff
@@ -33,23 +33,19 @@ function getTransactionsSections(data: OnyxTypes.SearchResults['data']): SearchT
         });
 }
 
-const searchTypeToItemMap = {
+const searchTypeToItemMap: SearchTypeToItemMap = {
     transaction: {
```

Comment:
is it possible to use here `[SEARCH_DATA_TYPES.TRANSACTION]: ...` instead of typing `transaction`?
it should work and it feels better, right?